CI-CD_DOCUMENTATION.md

```diff
@@ -154,5 +154,67 @@ NOTE: "closed" refers to "dismissed" alerts
 NOTE: **screenshots are only exemplary**
 
 #
+## Set up SonarCloud
+### SonarCloud is a cloud-based code quality and security service.
 
-Built with ❤ by [Pipeline Foundation](http://pipeline.foundation)
\ No newline at end of file
+#### Create your SonarCloud project.
+
+1. Go to https://sonarcloud.io/
```

Comment:
Please remove all numbering altogether and use bullets instead of numbers. Everywhere